extensions/vertx-http/deployment/src/test/resources/conf/http-permission-grant-config.properties

```diff
@@ -0,0 +1,42 @@
+quarkus.http.auth.basic=true
+quarkus.http.auth.policy.t1.roles-allowed=test,test2,admin,other
+quarkus.http.auth.policy.t1.grants-permissions.test=create-test
+quarkus.http.auth.policy.t1.grants-permissions.test2=create-test
+quarkus.http.auth.permission.t1.paths=/test/create
+quarkus.http.auth.permission.t1.policy=t1
+quarkus.http.auth.policy.t2.roles-allowed=test,test2,admin,other
+quarkus.http.auth.policy.t2.grants-permissions.test=list-test,update-test,delete-test:action1
+quarkus.http.auth.policy.t2.grants-permissions.test2=list-test,update-test:action2
+quarkus.http.auth.policy.t2.grants-permissions.admin=delete-test
+quarkus.http.auth.permission.t2.paths=/test/update,/test/list,/test/delete,/test/create2,/test/update2
+quarkus.http.auth.permission.t2.policy=t2
+quarkus.http.auth.policy.t3.roles-allowed=test,test2,admin
+quarkus.http.auth.policy.t3.grants-permissions.test=custom-test
```

Comment:
`grants-permissions` feels a bit inconsistent with how we say `roles-allowed`, I'd propose `permissions-allowed` or just `permissions` to make it shorter

Reply:
@sberyozkin It sounded strange to me, yeah, but I wanted to make it clear what it does :-) Personally I'd go with `permissions` in case there will be ever HTTP permissions allowed policy, it would be confusing to have `permissions-allowed` property in roles policy. I'll make it `permissions` later today. Thanks